action.yml

```
@@ -0,0 +1,19 @@
+name: 'pypi-publish'
+description: 'Upload Python distribution packages to PyPI'
+inputs:
+  user:
+    description: 'PyPI user'
+    required: false
+    default: __token__
+  password:
+    description: 'Password for your PyPI user'
+    required: true
+branding:
+  color: 'yellow'
+  icon: 'upload-cloud'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+    - ${{ inputs.user }}
```

Can we please not indent sequence items? I prefer it this way...
```suggestion
  - ${{ inputs.user }}
```

---

Without explicit config, `yamllint` complains about the indention if I don't indent it here.

---

Would you mind adding a `.yamllint` config with
```yaml
indentation:
  indent-sequences: false
```
then?

---

Nvmd, did it myself. I probably shouldn't bother you with style changes more than absolutely necessary :)